packages/aws-cdk-lib/aws-lambda/lib/adot-layers.ts

```diff
@@ -220,6 +220,11 @@ export class AdotLambdaLayerJavaSdkVersion extends AdotLambdaLayerVersion {
    */
   public static readonly V1_19_0 = new AdotLambdaLayerJavaSdkVersion('1.19.0');
 
+  /**
+   * Version 1.28.1
+   */
+  public static readonly V1_28_1 = new AdotLambdaLayerJavaSdkVersion('1.28.1');
```

Comment:
It's better to follow the convention of `AdotLambdaLayerPythonSdkVersion` and list the versions in descending order.